specification/monitor/resource-manager/Microsoft.Insights/preview/2020-05-01-preview/scheduledQueryRule_API.json

```diff
@@ -369,6 +373,11 @@
           "format": "duration",
           "description": "The period of time (in ISO 8601 duration format) on which the Alert query will be executed (bin size)."
         },
+        "OverrideQueryTimeRange": {
```

Comment:
This is a fix to s360 violation :
https://portal.azure-devex-tools.com/amekpis/correctness/detail?errorId=2EAB7F16-67CA-44F1-8877-39B8404F881F